### PR TITLE
content/docs: Update the build system documentation

### PR DESCRIPTION
**Pull request by cristian-vijelie**

### CODE REVIEW

File: content/en/docs/concepts/build-process.md [hunk 1]
```diff
@@ -1,40 +1,105 @@
...
   class="max-w-xl mx-auto"
   src="/assets/imgs/unikraft-overview.svg"
   title="Overview of the Unikraft build process."
-  caption="...."
-  alt="...."
   position="center"
 >}}
 
+The lifecycle of the construction of a Unikraft unikernel includes several
+distinct steps:
```
Comment (razvand) [inline content/en/docs/concepts/build-process.md:18]:
```suggestion
The lifecycle of the construction of a Unikraft unikernel includes several distinct steps:
```

File: content/en/docs/concepts/build-process.md [hunk 2]
```diff
@@ -1,40 +1,105 @@
...
+
 1. Configuring the Unikraft unikernel application with compile-time options;
-2. Fetching the remote "origin" code of libraries;
-3. Preparing the remote "origin" code of libraries;
-4. Compiling the libraries and the core Unikraft code; and,
-5. Finally, linking a final unikernel executable binary together.
+2. Fetching and preparing the source code for external libraries;
```
Comment (razvand) [inline content/en/docs/concepts/build-process.md:21]:
```suggestion
1. Fetching and preparing the source code for external libraries;
```

File: content/en/docs/concepts/build-process.md [hunk 3]
```diff
@@ -1,40 +1,105 @@
...
+3. Compiling the libraries and the core Unikraft code;
```
Comment (razvand) [inline content/en/docs/concepts/build-process.md:22]:
```suggestion
1. Compiling the libraries and the core Unikraft code;
```

File: content/en/docs/concepts/build-process.md [hunk 4]
```diff
@@ -1,40 +1,105 @@
...
+4. Linking the final unikernel image.
```
Comment (razvand) [inline content/en/docs/concepts/build-process.md:23]:
```suggestion
1. Linking the final unikernel image.
```

File: content/en/docs/concepts/build-process.md [hunk 5]
```diff
@@ -1,40 +1,105 @@
...
+  This type of configuration removes some configuration steps, but not all of them.
+
+* Writing the desired configuration in **kraft.yaml**.
+  The configuration gets loaded and the **.config** file is generated by running
+
+  ```
+  $ kraft configure
+  ```
+
+### Makefiles and config files
```
Comment (razvand) [inline content/en/docs/concepts/build-process.md:75]:
```suggestion
### Makefiles and .config Files
```

**Review by razvand — COMMENTED** [review]
This looks great. Thanks, @cristian-vijelie . I added some comments.

File: content/en/docs/concepts/build-process.md (hunk 2 above)
Comment (cristian-vijelie) [reply content/en/docs/concepts/build-process.md:21]: image

File: content/en/docs/concepts/build-process.md (hunk 3 above)
Comment (cristian-vijelie) [reply content/en/docs/concepts/build-process.md:22]: image

**Review by razvand — APPROVED** [review]
All good.

Reviewed-by: Razvan Deaconescu <razvan.deaconescu@cs.pub.ro>

**Review by nderjung — APPROVED** [review]
Thanks! :)

**Pull request merged**